server/data/prisonerContactRegistryApiClient.ts

```diff
@@ -14,14 +14,13 @@ export default class PrisonerContactRegistryApiClient {
     )
   }
 
-  async getPrisonerSocialContacts(approvedVisitorsOnly: boolean, offenderNo: string): Promise<Contact[]> {
+  async getPrisonersApprovedSocialContacts(offenderNo: string): Promise<Contact[]> {
```

Comment:
Removing the option to get 'unapproved' contacts because this is no longer needed: staff service only requests contacts directly from contact registry during booking journey which means we **always** want `approved`.